jekyll/_cci2/language-python.md

```diff
@@ -2,104 +2,266 @@
 layout: classic-docs
 title: "Language Guide: Python"
 short-title: "Python"
-description: "Building and Testing with Python on CircleCI 2.0"
+description: "Continuous integration and deployment with Python on CircleCI"
 categories: [language-guides]
 order: 7
 ---
 
-This document describes the [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file for a sample application written in Python. 
```

Comment:
Add back this link to the config ref (I just added all of these links from the user study data)